src/Main.hs

```diff
@@ -10,18 +9,24 @@ import Data.Aeson (decode)
 import Data.Aeson.Types
 import Data.ByteString.Lazy qualified as B
 import Data.Maybe
-import Data.Text
-import Database.SQLite.Simple (Connection, execute_, open)
+import Data.Text hiding (map)
+import Database.SQLite.Simple
 import GHC.Generics (Generic)
 import Ingredients (createIngredientsTables, handleCreateIngredient, handleGetIngredient, handleGetIngredients)
+import Recipe (createRecipeTable, handleAddRecipe, handleGetAllRecipes, handleGetRecipe)
 import User (createUser, handleLogin, handleRegister)
-import Web.Scotty
+import Web.Scotty (get, post, scotty)
+
+data Color = Color {peter :: String, jonas :: Int} deriving (Generic)
+
+instance ToJSON Color
```

Comment:
Das hab ich bewusst gelöscht, bitte nicht wieder hinzufügen.

Reply:
Removed in 61808d2e21d33e8bbea750823776691997934ac5